frontend/src/scenes/App.tsx

```diff
@@ -67,15 +66,14 @@ export function App(): JSX.Element | null {
     const { showApp, showingDelayedSpinner } = useValues(appLogic)
     const { user } = useValues(userLogic)
     const { currentTeamId } = useValues(teamLogic)
-    const { sceneConfig } = useValues(sceneLogic({ scenes: appScenes }))
```

Comment:
If you don't need the var here, can you replace this with `useMountedLogic(sceneLogic({ scenes }))` instead, as the `<LoadedSceneLogics />` below also uses sceneLogic, before AppScene. It'd be an endless game of cat and mouse, so just `useMountedValues` at the highest level will save the day.